### PR TITLE
Add requirements.txt for python packages.

### PR DESCRIPTION
**Pull request by blvrd**
@brainsik suggested that I add this from a conversation on [this ticket](https://www.pivotaltracker.com/story/show/163412116).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -76,14 +76,9 @@ RUN set -ex && cd ~ \
...
+ADD ./requirements.txt /tmp/requirements.txt
 RUN set -ex && cd ~ \
-  && sudo pip install --no-cache-dir --disable-pip-version-check \
-     awscli==1.16.100
-
-# install pre-commit
-RUN set -ex && cd ~ \
-  && sudo pip install --no-cache-dir --disable-pip-version-check \
-     pre-commit==1.14.2
+      && sudo pip install -r /tmp/requirements.txt --no-cache-dir --disable-pip-version-check
```
Comment (brainsik) [inline Dockerfile]: Even though this doesn't effect the layer size, it's good practice to clean up any temporary things needed during container build out:

```suggestion
      && sudo pip install -r /tmp/requirements.txt --no-cache-dir --disable-pip-version-check \
      && sudo rm -f /tmp/requirements.txt
```

**Review by brainsik — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (blvrd) [reply Dockerfile]: Gotcha.

**Comment by blvrd** [comment]
@brainsik is this good to go?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+awscli==1.16.100
```
Comment (brainsik) [inline requirements.txt]:
```suggestion
awscli==1.16.110
```

**Review by brainsik — APPROVED** [review]
Fix the `awscli` version (it was bumped since this PR got made) and you're good to go!

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+pre-commit==1.14.2
```
Comment (brainsik) [inline requirements.txt]:
```suggestion
pre-commit==1.14.4
```

**Review by brainsik — APPROVED** [review]
Also, please bump the pre-commit version. A new version came into Homebrew!

**Comment by blvrd** [comment]
These packages grow up so fast. Thanks 👍

**Pull request merged**